dev-libs/pigpio/files/pigpio-66-makefile.patch

```diff
@@ -0,0 +1,116 @@
+--- a/Makefile	2018-03-19 18:41:32.377835484 +0100
++++ b/Makefile	2018-03-19 20:53:05.366547745 +0100
+@@ -12,31 +12,31 @@
+ 
+ CFLAGS	+= -O3 -Wall -pthread
+ 
+-LIB1     = libpigpio.so
++LIB1     = libpigpio.so.0
+ OBJ1     = pigpio.o command.o
+ 
+-LIB2     = libpigpiod_if.so
++LIB2     = libpigpiod_if.so.0
+ OBJ2     = pigpiod_if.o command.o
+ 
+-LIB3     = libpigpiod_if2.so
++LIB3     = libpigpiod_if2.so.0
+ OBJ3     = pigpiod_if2.o command.o
+ 
+ LIB      = $(LIB1) $(LIB2) $(LIB3)
+ 
+ ALL     = $(LIB) x_pigpio x_pigpiod_if x_pigpiod_if2 pig2vcd pigpiod pigs
+ 
+-LL1      = -L. -lpigpio -pthread -lrt
++LL1      = -L. $(LIB1) -pthread -lrt
+ 
+-LL2      = -L. -lpigpiod_if -pthread -lrt
++LL2      = -L. $(LIB2) -pthread -lrt
+ 
+-LL3      = -L. -lpigpiod_if2 -pthread -lrt
++LL3      = -L. $(LIB3) -pthread -lrt
+ 
+-prefix = /usr/local
++prefix = /usr
+ exec_prefix = $(prefix)
+ bindir = $(exec_prefix)/bin
+ includedir = $(prefix)/include
+ libdir = $(prefix)/lib
+-mandir = $(prefix)/man
++mandir = $(prefix)/share/man
+ 
+ all:	$(ALL)
+ 
+@@ -64,41 +64,37 @@
+ 	$(CC) -o x_pigpiod_if2 x_pigpiod_if2.o $(LL3)
+ 
+ pigpiod:	pigpiod.o $(LIB1)
+-	$(CC) -o pigpiod pigpiod.o $(LL1)
+-	$(STRIP) pigpiod
++	$(CC) $(CFLAGS) $(LDFLAGS) -o pigpiod pigpiod.o $(LL1)
+ 
+ pigs:		pigs.o command.o
+-	$(CC) -o pigs pigs.o command.o
+-	$(STRIP) pigs
++	$(CC) $(CFLAGS) -o pigs pigs.o command.o
```

Comment:
Em… you need to do the same thing for all linked executables. Please don't expect reviewers to point all the places where you need to do something. We just point out the problems and you're expected to crticitally review your own work and find other problems like those listed.

Reply:
Ooops, i actually already had it for all binaries in there but got confused and removed it again as I somehow thought you pointed it out specifically for pigpiod as that is the only binary linking against libpigpio.so.0   .. sorry.. will push an update soon.